client/package.json

```diff
@@ -64,14 +64,14 @@
     "eslint-plugin-jest-dom": "^4.0.1",
     "eslint-plugin-react": "^7.27.1",
     "i18n-unused": "^0.8.0",
-    "jest": "^27.4.5",
+    "jest": "^28.1.0",
```

Comment:
When running tests now, I get the following error:
```
Test environment jest-environment-jsdom cannot be found. Make sure the testEnvironment configuration option points to an existing node module.
```

Can fix by adding `jest-environment-jsdom` to dev dependencies.

Reply:
Though all tests fail, even after that. 😩